packages/dev/core/src/ShadersWGSL/ShadersInclude/pbrFragmentDeclaration.fx

```diff
@@ -0,0 +1,240 @@
+uniform vEyePosition: vec4f;

```

Comment:
We don't need the non-UBO support fallback shaders in WGSL, as UBOs are always supported. So, these files can be removed:
* pbrFragmentDeclaration.fx
* pbrVertexDeclaration.fx
* lightFragmentDeclaration.fx
* lightVxFragmentDeclaration.fx

Reply:
great!! I was unsure. Thanks!